controllers/spifilecontentrequest_controller.go

```diff
@@ -135,23 +82,6 @@ func (r *SPIFileContentRequestReconciler) Reconcile(ctx context.Context, req ctr
 		return ctrl.Result{}, fmt.Errorf("failed to read the object: %w", err)
 	}
 
-	finalizationResult, err := r.finalizers.Finalize(ctx, &request)
```

Comment:
maybe it makes sense to cleanup finalizers from existing objects?
like here:
https://github.com/redhat-appstudio/service-provider-integration-operator/pull/716/files#diff-2c65b925cdcacc42c273f6ac66fc1fa7d51333eca6ab564ee1a100553ea4133bR122-R126

Reply:
agreed. done